packages/rule-components/src/ReportDetails/index.js

```diff
@@ -12,26 +12,35 @@ import ReactMarkdown from 'react-markdown';
 import { riskOfChangeMeta, totalRiskMeta } from './constants';
 import LinkInDetails from './LinkInDetails';
 import RiskDescription from './RiskDescription';
+import doT from 'dot';
 
 export { default as style } from './index.scss';
 
+const templateProcessor = (template, definitions) => {
+    const DOT_SETTINGS = { ...doT.templateSettings, varname: [ 'pydata' ], strip: false };
```

Comment:
No need to assign to a const
```JS
const templateProcessor = (template, definitions) =>(
  definitions ? doT.template(template, {
    ...doT.templateSettings,
    varname: [ 'pydata' ],
    strip: false
})(definitions) : template
);
```

Reply:
I like from time to time use constants, to make code more understandable, but ok, we need this changes as soon as possible